Source/Extensions/UIViewExtensions.swift

```diff
@@ -71,68 +71,52 @@ extension UIView {
 
 extension UIView {
 
-    func addBorder(with list: [SideBorder], thickness: Double, color: UIColor) {
+    func addBorder(with list: [UIRectEdge], thickness: Double, color: UIColor) {
```

Comment:
UIRectEdge is an `OptionSet`, which means you don't need to send it as an array

```suggestion
    func addBorder(with edges: UIRectEdge, thickness: Double, color: UIColor) {
```

You can try this in Xcode by pasting this code and seeing it compile :)
```swift
let edge1: UIRectEdge = [.left, .right]
let edge2: [UIRectEdge] = [.left, .right]
```

Furthermore, the fact this is an option set it will drastically impact our approach to add the borders. The whole approach should probably be changed to something like:

```swift
extension UIView {

  func applyBorder(_ style: ElementBorderStyle) {
    if style.edge.contains(.all) && style.corners.contains(.all) {
      // Apply standard border using relevant properties from provided style
      return
    }

    if style.edge.contains(.left) {
      // Add left edge line
    }

    // Perform this if check for each side and corner
  }

}
```